packages/integrations/solid/src/index.ts

```diff
@@ -44,6 +44,8 @@ function getViteConfiguration(isDev: boolean) {
 		},
 		ssr: {
 			external: ['babel-preset-solid'],
+			target: 'node',
```

Comment:
I don't think this config belongs in the Solid integration. If I'm understanding the config right, it controls whether the SSR build works in a webworker vs in node. This would be setting that for all Solid.js apps, but I think you're fixing a problem with Cloudflare, so this config should be in the Cloudflare integration.

Reply:
Or can Solid not work in webworker target? If that's the case I guess it would make sense to be here.

Reply:
> Or can Solid not work in webworker target? If that's the case I guess it would make sense to be here.

this is exactly the point, for solid to work in ssr mode it needs to target `node` (with target webworker vite is actually importing the browser exports and solid it is failing, my formerly wrong assumption was to fix that in the cloudflare adapter) overwriting the default ssr target `webwoker` for solid on the framework level is the correct thing to do.